ansible/playbooks/molecule/steamvr_docker/molecule.yml

```diff
@@ -0,0 +1,27 @@
+---
```

Comment:
please rename scenario to `install_steam_cmd_docker` or `install_steamvr_docker`